pkg/cmd/pipeline/start.go

```diff
@@ -167,7 +167,7 @@ like cat,foo,bar
 	)
 
 	c.Flags().StringSliceVarP(&opt.Labels, "labels", "l", []string{}, "pass labels as label=value.")
-	c.Flags().StringArrayVarP(&opt.Workspaces, "workspace", "w", []string{}, "pass the workspace.")
+	c.Flags().StringArrayVarP(&opt.Workspaces, "workspace", "w", []string{}, "pass one or more workspaces to map to the corresponding physical volumes as name=name,claimName=pvcName or name=name,emptyDir=")
```

Comment:
a lot of those could be as well gone to a common flag utility function ;)

Reply:
probably in a follow-up PR I can fix this